install_config/configuring_local.adoc

```diff
@@ -58,13 +58,19 @@ kubeletArguments:
      - PersistentLocalVolumes=true
 ----
 
+The `LocalPersistentVolumes` alpha feature also requires the `VolumeScheduling` alpha feature. This is a breaking change, and the following changes are required: 
+
+* The `VolumeScheduling` feature gate must also be enabled on kube-scheduler and kube-controller-manager components.
```

Comment:
Do we have any info on how to enable the feature gate?

Reply:
Do we have any info on how to enable the feature gate?

Reply:
I found this info on the Kubernetes reference site: https://kubernetes.io/docs/reference/feature-gates/

Should I add this link?